diffrax/_solver/srk.py

```diff
@@ -444,29 +348,25 @@ def make_zeros_aux(leaf):
         # Now the diffusion related stuff
         # Brownian increment (and space-time Lévy area)
         bm_inc = diffusion.contr(t0, t1, use_levy=True)
-        assert isinstance(bm_inc, self.minimal_levy_area), (
-            f"The diffusion term should be controlled by a Brownian path,"
-            f" initialised with"
-            f"`levy_area='{self.minimal_levy_area.__name__}'` or a subclass of it."
-            f"Got {bm_inc.__class__.__name__}."
-        )
+        assert isinstance(bm_inc, self.minimal_levy_area)
```

Comment:
Not a strong feeling -- it just struck me as a long error message for a situation that should in theory never arise. (Since this should already be handled by `_term_compatible`.) Put it back if you like.

Reply:
That's fair - now that we have a more principled `_term_compatible` it indeed shouldn't be necessary, I just thought it might catch anything that slips through the cracks. Should be fine though.